src/sqlacodegen/generators.py

```diff
@@ -615,6 +642,50 @@ def add_fk_options(*opts: Any) -> None:
 
         return render_callable(constraint.__class__.__name__, *args, kwargs=kwargs)
 
+    def _add_dialect_kwargs_and_info(
+        self, obj: Any, target_kwargs: dict[str, object], *, values_for_dict: bool
+    ) -> None:
+        """
+        Merge SchemaItem-like object's .info and .dialect_kwargs into target_kwargs.
+        - values_for_dict=True: keep raw values so pretty-printer emits repr() (for __table_args__ dict)
+        - values_for_dict=False: set values to repr() strings (for callable kwargs)
+        """
+        info_dict = getattr(obj, "info", None)
+        if info_dict:
+            target_kwargs["info"] = info_dict if values_for_dict else repr(info_dict)
+
+        dialect_keys: list[str]
+        try:
+            dialect_keys = sorted(getattr(obj, "dialect_kwargs"))
+        except Exception:
+            return
+
+        dialect_kwargs = getattr(obj, "dialect_kwargs", {})
+        for key in dialect_keys:
+            try:
+                value = dialect_kwargs[key]
+            except Exception:
+                continue
+            # Render values:
```

Comment:
Add a blank line after a control block ends
```suggestion

            # Render values:
```